docs/topics/modules/forking_example_repository.adoc

```diff
@@ -1,24 +1,22 @@
 [id="forking_example_repository"]
 = Forking an example repository
 
-In addition to creating projects based on quickstarts, you can also add projects by using the *Import an existing codebase* feature.
-
+In addition to creating projects using quickstarts, you can add an existing proeject to {osio} by importing the codebase.
 
 [discrete]
 == Prerequisites
 
-Reset your {osio} environment by following instructions in <<resetting_your_osio_account>>.
-
+* Ensure that you have reset your {osio} environment by following the instructions in <<resetting_your_osio_account>>.
```

Comment:
that you reset

Reply:
s/Ensure that you have reset/Reset 
This makes it more active.